mu4/scores/scoresmodule.cpp

```diff
@@ -48,6 +49,15 @@ void ScoresModule::registerExports()
     framework::ioc()->registerExport<IScoresConfiguration>(moduleName(), m_scoresConfiguration);
 }
 
+void ScoresModule::resolveImports()
+{
+    auto lr = framework::ioc()->resolve<framework::ILauncherUriRegister>(moduleName());
+    if (lr) {
+        lr->registerUri(Uri("musescore://scores/newscore"),
+                        framework::ContainerMeta(framework::ContainerType::QmlDialog, "MuseScore/Scores/NewScoreDialog.qml"));
```

Comment:
You can make using framework, then I think I fit in one line and it will be more readable

Reply:
fixed